docs/rest-api.md

```diff
@@ -31,7 +31,8 @@ Sample configuration:
         "jwt_secret_key": "somethingrandom",
         "CORS_origins": [],
         "username": "Freqtrader",
-        "password": "SuperSecret1!"
+        "password": "SuperSecret1!",
+        "ws_token": "sercet_Ws_t0ken"
```

Comment:
This ... concerns me a bit.
Maybe we should ensure that all our samples use the same `ws_token`.
as a seperate step, we can then do the same for the password parameter - and block usage of the "sample" values. (i'd see the actual check as a second step / seperate PR though - as it's currently not in place for `password` either).

Reply:
I'm not sure what you mean about the samples. Do you mean the `ws_token` in the example config should match the generated one below?

Reply:
"samples" is the example configuration / documentation examples.

if they all align - and use (for example) `sercet_Ws_t0ken` - we can check for that string - and reject it as "don't use demo samples". (that is however something for the future) ... it's just a thought for now ... but i think that'd make things a "bit" more secure by enforcing a "password change" of sorts

Reply:
Gotcha, that makes sense